src/data.tsx

```diff
@@ -0,0 +1,94 @@
+export const data_local = {
+
+  "data": {
+
+    "annual_salary":{
```

Comment:
I hadn't considered whether interfaces can be nested when defined, so this may be something we need to look into with typescript.